### DISCUSSION TITLE
sharing a directory from guest to host

### DISCUSSION THREAD

**Discussion by artincube** [utmapp/UTM#6406] [Linux]
I have found much info for the other way, share a host folder with the guest, but I need the other way round.
How can I do that?
Running Ubuntu ARM64 guest on Mac M1.
Thanks

**Comment by kpomichowski** [comment]
If you are using **VirtFS** as a sharing option, then specified read-write directory on the host and mounted on the guest can be used as a shared resource. It's the same directory viewed from the two machines. When you write some files in the already mounted directory in the guest, those files will appear as well in the host source directory that you wanted to share. No extra setup needed.
